projects/plugins/jetpack/extensions/store/membership-products/resolvers.js

```diff
@@ -54,7 +53,6 @@ const mapAPIResponseToMembershipProductsStoreData = ( response, registry, dispat
 	const postId = registry.select( editorStore ).getCurrentPostId();
 
 	dispatch( setConnectUrl( getConnectUrl( postId, response.connect_url ) ) );
-	dispatch( setShouldUpgrade( response.should_upgrade_to_access_memberships ) );
 	dispatch( setSiteSlug( response.site_slug ) );
 	dispatch( setUpgradeUrl( response.upgrade_url ) );
```

Comment:
Should this line go away too, since it seems the plan is to remove `upgrade_url` from the response as well?

Reply:
Thanks! Fixed in [2d92290](https://github.com/Automattic/jetpack/pull/30275/commits/2d922908ef672a095c1e18979bfc2877c343a9b9)